recipes/pato/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "pato" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jmeurisse/pato_releases_conda/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f4bf78d600f48f56382c93c758deb882b3600704080ae0a0942596405df1bef8
+  
+build:
+  number: 0
+  skip: True  # [win or linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - llvm-openmp
+    - make
+    - wget
+    - cmake 
+    - boost 
+    - metis
+    - cgal
+    - scotch
```

Dependencies that are linked to should be in `host`. `build` is only for build tools

```suggestion
  host:
    - boost 
    - metis
    - cgal
    - scotch
```

---

Also some of these may be needed in `run` as well

---

Thank you so much for your help! I’ll try your recommendations now

---

After making this change, the osx pipeline is still failing in the same spot